.vscode/settings.json

```diff
@@ -1,9 +1,20 @@
 {
-    "editor.formatOnSave": true,
+    "[python]": {
+        "editor.formatOnSaveMode": "file",
+        "editor.formatOnSave": true,
+        "editor.defaultFormatter": "eeyore.yapf",
```

Comment:
any particular reason that you chose the "eeyore.yapf" formatter? For the more famous [black formatter](https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter) I have seen inconvenience such as setting tab space, etc so I'd like to try out yapf as it seems to support indentation space, etc.

Reply:
According to https://github.com/google/yapf?tab=readme-ov-file#formatting-style yapf is preferred for google open source, though I’m not sure how official that is.

Reply:
Sounds good. I just needed the indentation customization so that formatter looks good to me.